contracts/multisig/src/msg.rs

```diff
@@ -33,6 +33,14 @@ pub enum ExecuteMsg {
     RegisterPublicKey {
         public_key: PublicKey,
     },
+    AuthorizeCaller {
+        // Adds an address to AUTHORIZED_CALLERS, representing the addresses that can start a signing session.
+        contract_address: Addr,
+    },
+    UnauthorizeCaller {
+        // Removes an address from AUTHORIZED_CALLERS
```

Comment:
Don't mention the internal data structure `AUTHORIZED_CALLERS` and move this comment above the message type